### ISSUE TITLE
MSVC not built on appveyor any more

### ISSUE THREAD

**Issue by GitMensch** [wbhart/mpir#271]
This was removed with 66019c2295b1e03f3a19c31dac001fa59a586055 - so [Appveyor CI](https://ci.appveyor.com/project/wbhart/mpir) only has MinGW-w64 builds. Was it added somewhere else?
[Travis CI](https://travis-ci.org/wbhart/mpir) has gcc and clang on "linux" (whatever this means) and osx.
Are the CI-environments used documented somewhere (I only found the NEWS file but it does not even mention the URLs)?

**Comment by wbhart** [comment]
I imagine Brian is doing Windows testing, rather than it being in the main repo. Perhaps @BrianGladman can comment on this?

It's certainly not documented. You can't document every little thing about a project and expect it to be kept up-to-date with zero volunteers.

**Comment by BrianGladman** [comment]
Correct.  I test on Windows and only maintain my own repository, which means that Windows builds in the main repository are not being maintained.  I only maintain builds with the Microsoft/Intel compilers and only do ongoing development on Windows x64  (although the win32 builds are still working as far as I know).
